councilmatic_core/management/commands/import_data.py

```diff
@@ -3090,3 +3084,21 @@ def executeTransaction(self, query, *args, **kwargs):
             trans.rollback()
             if raise_exc:
                 raise e
+
+    # Call this function when consolidating multiple queries into a single transaction!
+    # This function iterates over a list of queries and a list of params, and executes those queries.
+    # It, then, tries to commit the results of the execution and rolls back, if unable to do so.
+    def executeTransactionList(self, query_list, args_list, **kwargs):
+        trans = self.connection.begin()
```

Comment:
Is there a context handler we can use here?

Reply:
Yes, I added a `with` statement.